doc/classes/CodeEdit.xml

```diff
@@ -527,16 +527,16 @@
 	</methods>
 	<members>
 		<member name="auto_brace_completion_enabled" type="bool" setter="set_auto_brace_completion_enabled" getter="is_auto_brace_completion_enabled" default="false">
-			Sets whether brace pairs should be autocompleted.
+			If [code]true[/code], uses [member auto_brace_completion_pairs] to automatically insert the closing brace when the opening brace is inserted by typing or autocompletion. Also automatically removes the closing brace when using backspace on the opening brace.
 		</member>
 		<member name="auto_brace_completion_highlight_matching" type="bool" setter="set_highlight_matching_braces_enabled" getter="is_highlight_matching_braces_enabled" default="false">
-			Highlight mismatching brace pairs.
+			If [code]true[/code], highlights brace pairs when the caret is on either one using [member auto_brace_completion_pairs]. If matching, the pairs will be underlined. If a brace is unmatched, it is colored with [theme_item brace_mismatch_color].
 		</member>
 		<member name="auto_brace_completion_pairs" type="Dictionary" setter="set_auto_brace_completion_pairs" getter="get_auto_brace_completion_pairs" default="{ &quot;\&quot;&quot;: &quot;\&quot;&quot;, &quot;&apos;&quot;: &quot;&apos;&quot;, &quot;(&quot;: &quot;)&quot;, &quot;[&quot;: &quot;]&quot;, &quot;{&quot;: &quot;}&quot; }">
-			Sets the brace pairs to be autocompleted.
+			Sets the brace pairs to be autocompleted. The key is the the opening brace and the value is the closing brace that matches it. A brace is a [String] made of symbols. See [member auto_brace_completion_enabled] and [member auto_brace_completion_highlight_matching].
 		</member>
 		<member name="code_completion_enabled" type="bool" setter="set_code_completion_enabled" getter="is_code_completion_enabled" default="false">
-			Sets whether code completion is allowed.
+			If [code]true[/code], requests code completion when the [member ProjectSettings.input/ui_text_completion_query] input action is pressed. See [method _request_code_completion] or [signal code_completion_requested] to handle it.
```

Comment:
Again, just a matter of consistency.
```suggestion
			If [code]true[/code], requests code completion when the [member ProjectSettings.input/ui_text_completion_query] input action is activated. To handle it, see [method _request_code_completion] or [signal code_completion_requested].
```
or:
```suggestion
			If [code]true[/code], code completion can be requested when activating the [member ProjectSettings.input/ui_text_completion_query] action. To handle it, see [method _request_code_completion] or [signal code_completion_requested].
```
Or similar?

Reply:
How about

```suggestion
			If [code]true[/code], the [member ProjectSettings.input/ui_text_completion_query] action requests code completion. To handle it, see [method _request_code_completion] or [signal code_completion_requested].
```